README.md

```diff
@@ -156,13 +156,13 @@ Note: The theme build process is automatically triggered everytime your Lando co
 
 ### Quick Start
 
-1. Run `lando npm start`
+1. Run `lando yarn start`
 2. Go to `https://portlandor.lndo.site/pattern-lab`
 
 Here are some additional commands you may find useful.
 
-- Run `lando npm run build` to build both the theme assets and Pattern Lab.
-- Run `lando npm run watch` to build both the thme assets and  Pattern Lab, watch for changes and trigger rebuilds for both.
+- Run `lando yarn run build` to build both the theme assets and Pattern Lab.
```

Comment:
@RichardDavies , I went ahead and updated all the references to npm in the README.md.